### ISSUE TITLE
Firestore data is not showing up

### ISSUE THREAD

**Issue by ScriptedRain** [ScriptedRain/React-chat-app#3]
![image](https://user-images.githubusercontent.com/71009876/115836968-706b6e00-a45b-11eb-8018-cbf61fe86074.png)

There should be text from firestore showing up but it doesn't.
The issue could be that the app is not fully linked to the firebase yet

The changes should look like the ones in the video at the time 7:51
https://www.youtube.com/watch?v=2-LISBTczQE

@krishp058

**Comment by krishp058** [comment]
What line and what file is that in?

**Comment by ScriptedRain** [comment]
> What line and what file is that in?

The actions should take affect when the changes have been made in the `Channel.jsx` chech `app.js` too

**Comment by ScriptedRain** [comment]
also look at the video at 7:51 where the changes take place

**Comment by krishp058** [comment]
Ok ok thanks I will look into it, but I might of found to problem.

**Comment by krishp058** [comment]
Ok I looked into it, but I couldn't find the issue since i'm not fluent at JS yet, but have you tried re-watching the whole video to check if all the code is perfectly the same?

**Comment by krishp058** [comment]
Also could it be because of you not putting `;` at the end of the lines to state that the line has ended?

**Comment by bakhosRachwan** [comment]
Hello, I suggest you check in firebase that the collection name is identical to that you are calling here ".collection('messages')"
. Because the code looks correct to me.

**Comment by krishp058** [comment]
Yep I just checked, the collection name is still in fact called 'messages'. 
![Screenshot (5)](https://user-images.githubusercontent.com/74632222/115942851-888ccd00-a4ef-11eb-96f8-2b593df344b3.png)
 
Do you have any other idea that would fix this issue?

**Comment by krishp058** [comment]
After some google searches I found a way to display the messages.

I changed the following rule in firebase from:

`rules_version = '2';
service cloud.firestore {
  match /databases/{database}/documents {
    match /{document=**} {
      allow read, write: if false;
    }
  }
}`

to

`rules_version = '2';
service cloud.firestore {
  match /databases/{database}/documents {
    match /{document=**} {
      allow read, write: if true;
    }
  }
}`

Basically, I changed `allow read, write: if false` to  `allow read, write: if true`

![Messages](https://user-images.githubusercontent.com/74632222/115943964-592d8e80-a4f6-11eb-8b81-3c9837eeff3a.PNG)
